CHANGELOG.md

```diff
@@ -15,17 +15,18 @@ _Changes since 1.7.3_
 
 ### Web
 
--  Improves text input support in Safari on  mobile and desktop [#1941](https://github.com/JetBrains/compose-multiplatform-core/pull/1941)
--  Correct certain text input scenarios on Web targets
-New `WebTextInputService` seeks to achieve the following goals:
-   1. Add support to Safari (which happens to have its own flow of input events that are inconsistent with other browsers)
-   2. Correct behavior for composition and accent dialogue scenarios
-   3. Introduce some changes that we will use for improving accessibility
-- `skiko.js` is redundant in case of K/Wasm Compose Multiplatform for web applications and it can be removed from index.html files to not load redundant files. `skiko.js` will be removed from the k/wasm distribution in the future releases. `skiko.js` is still needed in case of K/JS Compose Multiplatform for web apps [#5134](https://github.com/JetBrains/compose-multiplatform/pull/5134)
+- Improves text input support in Safari on  mobile and desktop [#1941](https://github.com/JetBrains/compose-multiplatform-core/pull/1941)
+- Correct certain text input scenarios on Web targets [#1941](https://github.com/JetBrains/compose-multiplatform-core/pull/1941)
 
 ### Resources
 
-- Add FontVariation.Settings support to the resources library [#5183](https://github.com/JetBrains/compose-multiplatform/pull/5183)
+- Add `FontVariation.Settings` support to the resources library [#5183](https://github.com/JetBrains/compose-multiplatform/pull/5183)
+
+## Known Issues
+
+### iOS
+
+- Back gesture may remain stuck in the middle, the fix will be available in the next pre-release version [#1861](https://github.com/JetBrains/compose-multiplatform-core/pull/1861)
```

Comment:
Next prerelease version? I guess it's already fixed and should not be a part of final changelog